pytest.ini

```diff
@@ -16,7 +16,11 @@ markers =
     tpu_only: marks tests to be run on TPUs only
     gpu_only: marks tests to be run on GPUs only
     cpu_only: marks tests to be run on CPUs only
+    decoupled: tests that validate offline / DECOUPLE_GCLOUD=TRUE mode
```

Comment:
can you comment that this marker is not to be used manually -- it is applied automatically when the test is not marled as external_*  (if my understanding is correct)